frontend/src/components/FavoritesPage/Favorite.jsx

```diff
@@ -54,6 +54,22 @@ function Favorite() {
   return (
 
     <div className="favorite-container">
+            <div className='light x1'></div>
+            <div className='light x2'></div>
+            <div className='light x3'></div>
+            <div className='light x4'></div>
+            <div className='light x5'></div>
+            <div className='light x6'></div>
+            <div className='light x7'></div>
+            <div className='light x8'></div>
+            <div className='light x9'></div>
+            <div className="bg-animation">
+            <div id="stars1"></div>
+            <div id="stars2"></div>
+            <div id="stars3"></div>
+            <div id="stars4"></div>
+            <div id="stars5"></div>
+            </div>
```

Comment:
RECOMMENDED: Consider extracting this as a component since there's lots of repetitions.